ui/src/app/components/runtime-configuration-panel/customize-panel-footer.tsx

```diff
@@ -50,7 +50,7 @@ export const CustomizePanelFooter = ({
     (gcePersistentDisk.size > analysisConfig.diskConfig.size ||
       gcePersistentDisk.diskType !== analysisConfig.diskConfig.detachableType);
 
-  return unattachedPdExists ? (
+  return unattachedPdExists && !runtimeExists ? (
```

Comment:
for GCE, an unattached PD means there is no runtime, so it's appropriate to only show PD options here.

for DataProc, the runtime can exist with an unattached PD (because it's for GCE) so we need this condition to be false.